mods/seasons/manual/info_for_map_makers.md

```diff
@@ -42,17 +36,16 @@ The mask is a new foliage layer named `ssSnowMask`. Everywhere you paint, snow w
 
 There are also some other layers that we use for other purposes or for the future.
 
-- Copy the three texture files invisible_mask_diffuse, invisible_mask_diffuse0, and seasons_density into your map01 folder.
-- The code to use is also supplied in the map01.i3d file.
+- Download invisible_mask_diffuse, invisible_mask_diffuse0, and seasons_density from https://github.com/RealismusModding/FS17_seasons/tree/develop/modding/snowMask/map01 and put them into your map01 folder.
```

Comment:
Sounds good, I will try to change to that kind of direct links. So hold of committing the pull request a few minutes.